packages/modular-scripts/src/check/verifyWorkspaceDependencies.ts

```diff
@@ -29,13 +29,6 @@ export async function check(target?: string): Promise<boolean> {
       valid = false;
     }
 
-    if (!packageInfo.location.startsWith('packages')) {
```

Comment:
Shouldn't we at least limit it to packages found in the `workspaces` field in `package.json`? This would keep a level of compatibility with yarn. Yarn implements that boundary.

Won't it potentially pick up stuff in e.g. `__tests__` with zero restriction?

Reply:
I don't think so; we get our workspaces through `getWorkspaceInfo`, which in turn uses the workspace resolver to scan all the workspaces in the repo, looking at the `workspaces` field in `package.json`. As a consequence, we can't get a workspace outside `workspaces`, since we use `workspaces` in the first place to generate their list.